Dockerfile.raspberry-pi2

```diff
@@ -1,14 +1,18 @@
 # assumes at least ARMv71 but not ARMv8 (aarch64) yet
-FROM balenalib/raspberry-pi2-debian-node:latest-buster-build
+FROM balenalib/raspberry-pi2-debian:buster-build
 
-RUN [ "cross-build-start" ]
```

Comment:
just to make sure, does it work? Travis is arm64

Reply:
Yes, the emulated ones would not work only because the `cross-build-end` command was run before any of the real tests started so I removed them as they were no longer necessary.

Reply:
ok